src/client/components/app-core/text.css

```diff
@@ -17,3 +17,17 @@ figcaption {
   color: var(--text-light-color);
   font-style: italic;
 }
+
+code,
+pre {
+  background-color: var(--neutral-color);
+  border-radius: 3px;
```

Comment:
border-radius: 3px now has been used twice ;)
Should we use a variable for this?